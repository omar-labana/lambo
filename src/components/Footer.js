const Footer = () => {
    return (
        <footer className="flex items-center justify-around py-20 text-white border-t border-yellow mt-10">
            <div>
                <h3 className="yellow text-xl">Location</h3>
                <p>
                    4321 Rose Vally, EG
                </p>
            </div>
            <div>
                <h3 className="yellow text-xl">Working Hours</h3>
                <p>
                    Mondat - Friday 10 am - 9pm
                </p>
            </div>
            <div>
                <h3 className="dancing-font text-7xl">Lambo</h3>
                <p>
                    info@lambo.lambo
                </p>
            </div>
        </footer>
    )
}

export default Footer
