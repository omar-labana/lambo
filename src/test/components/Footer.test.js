import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('Footer Component', () => {
  const wrapper = shallow(<Footer />);

  it('exists', () => {
    expect(wrapper.find('Footer')).toBeTruthy();
  });

  it('has to match snap', () => {
    const home = renderer.create(<Footer />);
    expect(home).toMatchSnapshot();
  });
});
