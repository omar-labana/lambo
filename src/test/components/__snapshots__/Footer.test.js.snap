// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component has to match snap 1`] = `
<footer
  className="flex items-center justify-around py-20 text-white border-t border-yellow mt-10"
>
  <div>
    <h3
      className="yellow text-xl"
    >
      Location
    </h3>
    <p>
      4321 Rose Vally, EG
    </p>
  </div>
  <div>
    <h3
      className="yellow text-xl"
    >
      Working Hours
    </h3>
    <p>
      Mondat - Friday 10 am - 9pm
    </p>
  </div>
  <div>
    <h3
      className="dancing-font text-7xl"
    >
      Lambo
    </h3>
    <p>
      info@lambo.lambo
    </p>
  </div>
</footer>
`;
